manifests/init.pp

```diff
@@ -71,77 +62,43 @@
 #                               defaults to number of processors and maxs at 8
 #
 class pulp (
-
-  $oauth_key = $pulp::params::oauth_key,
-  $oauth_secret = $pulp::params::oauth_secret,
-
-  $messaging_url = $pulp::params::messaging_url,
-  $messaging_ca_cert = $pulp::params::messaging_ca_cert,
-  $messaging_client_cert = $pulp::params::messaging_client_cert,
-
-  $broker_url = $pulp::params::broker_url,
-  $broker_use_ssl = $pulp::params::broker_use_ssl,
-
-  $consumers_ca_cert = $pulp::params::consumers_ca_cert,
-  $consumers_ca_key = $pulp::params::consumers_ca_key,
-  $ssl_ca_cert = $pulp::params::ssl_ca_cert,
-
-  $consumers_crl = $pulp::params::consumers_crl,
-
-  $ssl_ca_cert = $pulp::params::ssl_ca_cert,
-
-  $default_password = $pulp::params::default_password,
-
-  $repo_auth = true,
-
-  $reset_data = false,
-  $reset_cache = false,
-
-  $qpid_ssl = $pulp::params::qpid_ssl,
-  $qpid_ssl_cert_db = $pulp::params::qpid_ssl_cert_db,
-  $qpid_ssl_cert_password_file = $pulp::params::qpid_ssl_cert_password_file,
-
-  $proxy_url      = $pulp::params::proxy_url,
-  $proxy_port     = $pulp::params::proxy_port,
-  $proxy_username = $pulp::params::proxy_username,
-  $proxy_password = $pulp::params::proxy_password,
-
-  $num_workers = $pulp::params::num_workers,
-
-  ) inherits pulp::params {
+  $oauth_key                   = $pulp::params::oauth_key,
+  $oauth_secret                = $pulp::params::oauth_secret,
+  $messaging_url               = $pulp::params::messaging_url,
+  $messaging_ca_cert           = $pulp::params::messaging_ca_cert,
+  $messaging_client_cert       = $pulp::params::messaging_client_cert,
+  $broker_url                  = $pulp::params::broker_url,
+  $broker_use_ssl              = $pulp::params::broker_use_ssl,
+  $consumers_ca_cert           = $pulp::params::consumers_ca_cert,
+  $consumers_ca_key            = $pulp::params::consumers_ca_key,
+  $consumers_crl               = $pulp::params::consumers_crl,
+  $ssl_ca_cert                 = $pulp::params::ssl_ca_cert,
+  $default_password            = $pulp::params::default_password,
+  $repo_auth                   = true,
+  $reset_data                  = false,
+  $reset_cache                 = false,
+  $proxy_url                   = $pulp::params::proxy_url,
+  $proxy_port                  = $pulp::params::proxy_port,
+  $proxy_username              = $pulp::params::proxy_username,
+  $proxy_password              = $pulp::params::proxy_password,
+  $num_workers                 = $pulp::params::num_workers,
+) inherits pulp::params {
 
   include ::apache
+  include ::apache::mod::wsgi
+  include ::apache::mod::ssl
 
-  if (versioncmp($::mongodb_version, '2.6.5') >= 0) {
-    $mongodb_pidfilepath = '/var/run/mongodb/mongod.pid'
-  } else {
-    $mongodb_pidfilepath = '/var/run/mongodb/mongodb.pid'
-  }
+  include ::mongodb::globals
```

Comment:
If you're not overriding any globals, there's no point in including it since ```mongodb::params``` inherits from it.